MFVNeutralino/plugins/EventHistos.cc

```diff
@@ -2,799 +2,1243 @@
-        TH1F* h_l1_bits;
-
-        TH1F* h_npu;
-
-        TH1F* h_bsx;
-        TH1F* h_bsy;
-        TH1F* h_bsz;
-        TH1F* h_bsphi;
-
-        TH1F* h_npv;
-        TH1F* h_pvx;
-        TH1F* h_pvy;
-        TH1F* h_pvxwide;
-        TH1F* h_pvywide;
-        TH1F* h_pvz;
-        TH1F* h_pvcxx;
-        TH1F* h_pvcxy;
-        TH1F* h_pvcxz;
-        TH1F* h_pvcyy;
-        TH1F* h_pvcyz;
-        TH1F* h_pvczz;
-        TH1F* h_pvrho;
-        TH1F* h_pvrhowide;
-        TH1F* h_pvphi;
-        TH1F* h_pvntracks;
-        TH1F* h_pvscore;
-        TH1F* h_pvsx;
-        TH1F* h_pvsy;
-        TH1F* h_pvsxwide;
-        TH1F* h_pvsywide;
-        TH1F* h_pvsz;
-        TH1F* h_pvsrho;
-        TH1F* h_pvsrhowide;
-        TH1F* h_pvsphi;
-        TH1F* h_pvsscore;
-        TH1F* h_pvsdz;
-        TH1F* h_pvsmindz;
-        TH1F* h_pvsmaxdz;
-        TH1F* h_pvsmindz_minscore;
-        TH1F* h_pvsmaxdz_minscore;
-        TH1F* h_pvbs_dist;
-
-        TH1F* h_njets;
-        TH1F* h_njets20;
-        TH1F* h_ncalojets;
-        TH1F* h_ncalojets40;
-        TH1F* h_nhltcalojets;
-        TH1F* h_nhltcalojets40;
-        TH1F* h_nhltbjets;
-        TH1F* h_nhltcalobjets;
-        TH1F* h_nhltcalobjets_low;
-        static const int MAX_NJETS = 10;
-        TH1F* h_jet_pt[MAX_NJETS+1];
-        TH1F* h_jet_eta[MAX_NJETS+1];
-        TH1F* h_jet_phi[MAX_NJETS+1];
-        TH1F* h_jet_nseedtrack[MAX_NJETS+1];
-
-        TH1F* h_jet_energy;
-        TH1F* h_jet_ht;
-        TH1F* h_jet_ht_40;
-        TH2F* h_jet_calojet_ht;
-        TH2F* h_jet_calojet_ht_40;
-        TH1F* h_jet_diagnostics_lo;
-        TH1F* h_jet_diagnostics_hi;
-
-        TH1F* h_calojet_pt[MAX_NJETS+1];
-        TH1F* h_calojet_eta[MAX_NJETS+1];
-        TH1F* h_calojet_phi[MAX_NJETS+1];
-        TH2F* h_calojet_i_pt;
-        TH1F* h_calojet_ht_30;
-        TH1F* h_calojet_ht_40;
-        TH1F* h_calojet_diagnostics_lo;
-        TH1F* h_calojet_diagnostics_hi;
-
-        TH1F* h_jet_pairdphi;
-        TH1F* h_jet_pairdr;
-
-        TH1F* h_min_hltcalo_pfjet_dr;
-
-        TH1F* h_met;
-        TH1F* h_metphi;
-        TH1F* h_metnomu;
-        TH1F* h_metnomuphi;
-
-        TH1F* h_nbtags[3];
-        TH2F* h_nbtags_v_bquark_code[3];
-        TH1F* h_jet_bdisc_csv;
-        TH1F* h_jet_bdisc_deepcsv;
-        TH1F* h_jet_bdisc_deepflav;
-        TH2F* h_jet_bdisc_deepflav_v_bquark_code;
-        TH1F* h_bjet_pt;
-        TH1F* h_bjet_eta;
-        TH1F* h_bjet_phi;
-        TH1F* h_bjet_energy;
-        TH1F* h_bjet_pairdphi;
-
-        TH1F* h_thresh_csvtags;
-        TH1F* h_thresh_hardcsvtags;
-
-        TH1F* h_nmuons[2];
-        TH1F* h_nelectrons[2];
-        TH1F* h_nleptons[2];
-
-        TH1F* h_leptons_pt[2][2];
-        TH1F* h_leptons_eta[2][2];
-        TH1F* h_leptons_phi[2][2];
-        TH1F* h_leptons_dxy[2][2];
-        TH1F* h_leptons_dxybs[2][2];
-        TH1F* h_leptons_dz[2][2];
-        TH1F* h_leptons_iso[2][2];
-
-        TH1F* h_n_reljettks;
```

Comment:
Are we sure about all of these histograms we're deleting, particularly the calojet and btag ones? @pkotamnives @abbywarden @shaunhogan